get_ioc_usage.py

```diff
@@ -56,9 +56,12 @@ def print_instruments_with_ioc(instrument_configs, ioc_name):
     for instrument, iocs in instrument_configs.items():
 
         for ioc in iocs:
-            if ioc.lower().startswith(ioc_name.lower()):
-                print(instrument)
+            ioc_lower = ioc.lower()
+            if ioc_lower == ioc_name.lower():
+                print(f"{instrument}")
                 break
+            elif ioc_lower.startswith(ioc_name.lower()):
+                print(f"{instrument} has {ioc}")
 
```

Comment:
What is the sort order of iocs ? If for example you gave the ioc_name as "ITC5" and there were "ITC5" and "ITC500" iocs on the system then I wasn't sure if you would just get "ITC5" or both "ITC5" and "ITC500" printed.

Reply:
You're right, this is overly complex for no good reason, I'll just print everything